config/config.php

```diff
@@ -22,5 +22,9 @@
         'attempts' => 10,
         // Time to wait (seconds)
         'time_to_wait' => 300,
+        // In case you need to setup a mechanism to whitelist someone
+        'is_whitelisted' => function() {
```

Comment:
well now it's odd not to provide `is_throttled` (which would return `$failures >= \Arr::get($wait_failures_config, 'attempts', 3)`)